2차/류동준_정수-제곱근-판별.py

```diff
@@ -0,0 +1,9 @@
+def check(n):
+    value = pow(n,0.5)
+    if int(value) == value:
+        return pow(value+1,2)
+    else:
+        return -1
+def solution(n):
+    value = check(n)
+    return value
```

Comment:
보통 값을 저장 후에 반환하던가요? (개인적인 질문)
```suggestion
    return check(n)
```